packages/caravan-wallets/src/bcur2/__tests__/utils.test.ts

```diff
@@ -0,0 +1,190 @@
+import { Network } from "@caravan/bitcoin";
+import { CryptoAccount, CryptoHDKey } from "@keystonehq/bc-ur-registry";
+import { vi, describe, it, expect, beforeEach } from "vitest";
+
+import { processCryptoAccountCBOR, processCryptoHDKeyCBOR } from "../utils";
+
+// Valid compressed public key for testing
+const testKey = Buffer.from(
+  "0279BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798",
+  "hex"
+);
+const testChain = Buffer.from(
+  "000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f",
+  "hex"
+);
+
+// Mock the bc-ur-registry module
+vi.mock("@keystonehq/bc-ur-registry", () => {
+  class MockComponent {
```

Comment:
I think you might be able to vitest-mock-extended for these but I don't know for sure.